src/Uno.Extensions.Maui.UI/MauiEmbedding.cs

```diff
@@ -126,7 +126,12 @@ private static void InitializeApplicationMainPage(IApplication iApp)
 			return;
 		}
 
+#if ANDROID
+		var services = app.Handler.MauiContext.Services;
+		var context = new MauiContext(services, services.GetRequiredService<Android.App.Activity>());
+#else
 		var context = app.Handler.MauiContext;
+#endif
```

Comment:
I'm not sure if the intent is specifically overriding `context` in this code path only, or if there are other places that might be accessing `app.Handler.MauiContext` that needs the android context?, in which case maybe we can do `app.Handler.SetMauiContext`?

Reply:
Yeh there's a wider refactor that's required for this code - this is a stopgap to get things working in latest template